### ISSUE TITLE
Vieuws based on http urls that respond according to a CSV schema

### ISSUE THREAD

**Issue by GoogleCodeExporter** [runt18/google-bigquery#162]
```
It would be great if I could build a "view" in appEngine (or any other 
webserver) that would respond to a specific url request. Of course that 
response should confirm to the schema definition of the view.

If I could do this I could use data stored in GCD without a job to load data 
into BQ. 

It would be nice if the url would be requested only once per query on the view.
```

Original issue reported on code.google.com by `sjuul.ja...@transceptor-technology.com` on 7 Oct 2014 at 1:42

**Comment by GoogleCodeExporter** [comment]
```
I'm afraid I don't understand this request.  Can you elaborate on what you're 
asking for here?

Are you wanting to query over data in Google Cloud Storage without having to 
import it into BigQuery?  If so, that is now supported : 
https://cloud.google.com/bigquery/federated-data-sources
```

Original comment by `thomasp...@google.com` on 25 Nov 2015 at 1:14
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Marking fixed based federated data as mentioned in #1.
```

Original comment by `jcon...@google.com` on 1 Mar 2016 at 7:45
- Changed state: **Fixed**
